pumps/stdout.go

```diff
@@ -0,0 +1,100 @@
+package pumps
+
+import (
+	"context"
+	"github.com/TykTechnologies/logrus"
+	"github.com/TykTechnologies/tyk-pump/analytics"
+	"github.com/TykTechnologies/tyk-pump/logger"
+	"github.com/mitchellh/mapstructure"
+)
+
+var stdOutPrefix = "stdout-pump"
+
+type StdOutPump struct {
+	CommonPumpConfig
+	conf *StdOutConf
+}
+
+type StdOutConf struct {
+	Format       string `mapstructure:"format"`
+	LogFieldName string `mapstructure:"log_field_name"`
+}
+
+func (s *StdOutPump) GetName() string {
+	return "Stdout Pump"
+}
+
+func (s *StdOutPump) New() Pump {
+	newPump := StdOutPump{}
+	return &newPump
+}
+
+func (s *StdOutPump) Init(config interface{}) error {
+
+	s.conf = &StdOutConf{}
+	err := mapstructure.Decode(config, &s.conf)
+
+	if err != nil {
+		log.WithFields(logrus.Fields{
+			"prefix": statsdPrefix,
+		}).Fatal("Failed to decode configuration: ", err)
+	}
+
```

Comment:
need to parse env vars here. Take a look at the master csv pump https://github.com/TykTechnologies/tyk-pump/blob/c9d9b92ecea143423e40acdfa5e2d3d71b5bb63d/pumps/csv.go#L52